tests/xfstests/run.pm

```diff
@@ -528,7 +534,16 @@ sub run {
 
     heartbeat_start if $enable_heartbeat == 1;
 
+    # Generate xfstests blacklist and softfail list
     my %black_list = (generate_xfstests_list($BLACKLIST), exclude_grouplist);
+    if (my $issues = get_var('XFSTESTS_KNOWN_ISSUES')) {
```

Comment:
I'm puzzled when checking this parameter in verification run, it's define to XFSTESTS_KNOWN_ISSUES=http://127.0.0.1:86/test3.json. And I can't see this file by clicking the URL. Then how do I know which tests contain in this list, and how to maintain this file?

Reply:
New verification run: http://10.67.133.133/tests/659